### DISCUSSION TITLE
the get_value() has no .Text to get the text from LocalizedText

### DISCUSSION THREAD

**Discussion by wgoebl** [FreeOpcUa/opcua-asyncio#1233] [Q&A]
how can I extract the Text from the LocalizedText when I'm using the get_value() ?
E.g.: I want to get the: 
    PREPARED_FOR_UPDATE 
from: LocalizedText(Encoding:3, Locale:en-us, Text:PREPARED_FOR_UPDATE)

I've used these with [python-opcua](https://github.com/FreeOpcUa/python-opcua) and now I want to use a method in asyncua.

**Comment by AndreasHeine** [comment]
just access the class prop with ".Text"
